src/components/Profile/Profile.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import s from "./Profile.module.css";
+import MyPosts from "./MyPosts/MyPosts";
+
+const ProfileLink = "https://animal-wallpaper.com/wallpaper/minimalist-background-hd-For-Background-HD-Wallpaper.jpg"
```

Comment:
Consider changing the case of "ProfileLink" to upper case like so "PROFILE_LINK".

Reply:
complite